mozci/console/commands/push.py

```diff
@@ -28,16 +30,52 @@ def handle(self):
 
 class ClassifyCommand(Command):
     """
-    Display the classification for a given push as GOOD, BAD or UNKNOWN.
+    Display the classification for a given push (or a range of pushes) as GOOD, BAD or UNKNOWN.
 
     classify
         {branch=mozilla-central : Branch the push belongs to (e.g autoland, try, etc).}
+        {--rev= : Head revision of the push.}
+        {--from-date= : Lower bound of the push range (as a date in yyyy-mm-dd format).}
+        {--to-date= : Upper bound of the push range (as a date in yyyy-mm-dd format).}
         {--medium-confidence=0.8 : Medium confidence threshold used to classify the regressions.}
         {--high-confidence=0.9 : High confidence threshold used to classify the regressions.}
         {--output= : Path towards a directory to save a JSON file containing classification and regressions details in.}
     """
 
     def handle(self):
+        branch = self.argument("branch")
+
+        if not self.option("rev"):
+            from_date = self.option("from-date")
+            to_date = self.option("to-date")
+            if not from_date or not to_date:
```

Comment:
Thanks for the validation! I think we should put this check outside of the `if not self.option("rev"):` statement though. Otherwise if the user specifies both `--rev` and `--to-date` we'll never reach this validation.

Reply:
From my own understanding, the user should be able to only provide either a revision OR both from and to dates. When the revision is explicitly given, we only want to classify it and nothing else. But it really depends and what you want to achieve and it should be easily refactorable if you want me to support alternative cases.

Reply:
Yeah, that's correct. I just meant for the validation. Right now if the user passes `--rev <rev> --to-date <date> --from-date <date>`, we'll silently ignore the `--to-date` and `--from-date` since this validation is happening inside the if statement.

I was just asking to move the validation up higher so the above scenario becomes an error.

Reply:
Ok, thx for the explanation!